### PR TITLE
OPENE-457 templates moved to module

### PR DESCRIPTION
**Pull request by ArnasPetraitis**

### CODE REVIEW

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
r

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

**Comment by ArnasPetraitis** [comment]
retest this please

File: openesdh-doc-templates/src/main/amp/module.properties [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+# SDK Sample module
+
+# ==== Beginning of Alfresco required/optional properties ====== #
+# NB: These properties are filtered at build time by Maven, single
+# sourcing from POM properties  
+module.id=${project.artifactId}
+#module.aliases=myModule-123, my-module
```
Comment (jrudinskij) [inline openesdh-doc-templates/src/main/amp/module.properties]: We need to set proper alias and openesdh-repo dependency to make the module settable for tenants.
See the "OpenE modules management in multi-tenancy environment" section of the https://collaborate.consulting.visma.com/display/OPENE/OpenE+Multi-Tenancy+Implementation

File: openesdh-doc-templates/src/main/java/dk/openesdh/doctemplates/services/documenttemplate/DocumentTemplateServiceImpl.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package dk.openesdh.repo.services.documents;
```
Comment (jrudinskij) [inline openesdh-doc-templates/src/main/java/dk/openesdh/doctemplates/services/documenttemplate/DocumentTemplateServiceImpl.java:1]: I guess we need some sort of security proxies for document templates services to check whether a tenant has permissions to call services. See the TenantOpeneModuleService.checkCurrentTenantModuleEnabled()

**Pull request merged**